tokio/src/io/util/empty.rs

```diff
@@ -50,16 +50,18 @@ impl AsyncRead for Empty {
     #[inline]
     fn poll_read(
         self: Pin<&mut Self>,
-        _: &mut Context<'_>,
+        cx: &mut Context<'_>,
         _: &mut ReadBuf<'_>,
     ) -> Poll<io::Result<()>> {
+        let _ = ready!(poll_proceed_and_make_progress(cx));
```

Comment:
You're just ignoring a `()` here, no?
```suggestion
        ready!(poll_proceed_and_make_progress(cx));
```

Reply:
ah you're right, I was not very aware of when people assign to `_` vs ignore the return value of functions. I understand now that people assign `Result` to `_` because it's must use. Will fix right now